cli.py

```diff
@@ -34,14 +34,13 @@
 @click.option('-a', is_flag=True, help="Add this flag if you want to include analytics page for nodes")
 @click.option('-r', is_flag=True, help="Add this flag if you want to include registrations page for nodes")
 @click.option('-k', is_flag=True, help="Add this flag if you want to include forks page for nodes")
-def cli_entry_point(scrape, resume, verify, resume_verify, compile_active, delete, dm, tf, rn, ctf, registrations, users,
-                    institutions, nodes, d, f, w, a, r, k):
-    if scrape and resume and verify and compile_active:
-        click.echo('Invalid parameters.')
-        return
+def cli_entry_point(scrape, resume, verify, resume_verify, compile_active, delete, dm, tf, rn, ctf, registrations,
+                    users, institutions, nodes, d, f, w, a, r, k):
 
-    if not scrape and not resume and not verify and not resume_verify and not compile_active and not delete:
-        click.echo('You have to choose a mode to run')
+    # Check to see if more than one option is chosen.
+    if sum(map(bool, [scrape, resume, verify, resume_verify, compile_active, delete])) != 1:
```

Comment:
Um... I map it to bool to make sure that no one would attempt to break it by passing a non-boolean parameter. Defensive programming.